src/style.css

```diff
@@ -0,0 +1,103 @@
+* {
+  margin-top: 0;
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 300;
+  src: url(./fonts/Avenir.ttc) format("truetype"),
+    url(./fonts/Avenir-Book.ttf) format("truetype"),
+    url(./fonts/Avenir-Heavy.ttf) format("truetype");
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+}
+
+.search-form {
+  display: flex;
+  justify-content: center;
+  margin: 20px auto 0;
+  width: 100%;
+  height: 70px;
+}
+
+.secondSearchBar {
+  height: 42px;
+}
+
+.search-form__label {
+  position: relative;
+  display: block;
+  width: 100%;
+}
+
+.search-form__input {
+  width: 100%;
+  height: 100%;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  color: #3d4e61;
+  font-weight: 300;
+  font-family: inherit;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  outline: none;
+}
+
+.search-form__input:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.inputBig {
+  padding-left: 62px;
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.inputSmall {
+  padding-left: 33px;
+  font-size: 14px;
+  line-height: 19px;
+}
```

Comment:
У нас немає ніякого проміжного search, відповідно, чому має бути великий і малий search? У нас є search-input та search-input--small (--small - модифікатор). Тому всі стилі які прописані для “великого” інпуту ми задаємо для всіх інпутів в цілому, а за допомогою модифіктора small міняємо тільки потрібні значення, щоб наш інпут став малим.

Reply:
Тут в мене виникли складнощі.. 

Коли я прибрала зайвий CSS код (на модифікатор Big), задавши лиш загальний для всіх інпутів і потім коригуючи за допомогою Small і хотіла прибрати зайвий search-input-Big в HTML, то в мене не проходить тест з текстом:

"ENGINE ERROR: Navigation failed because browser has disconnected!" на "label: Big search bar data-qa_big selector: [data-qa="big"]
filename: Big_search_bar_data-qa_big_1024px.png
url: [test](http://localhost:3001/index.html)[reference](https://mate-academy.github.io/layout_solutions/search-bar-airbnb/)diff%: 22.93 diff-x: -776 diff-y: 110"

Не впевнена, що скрін цей відобразився...
<img width="1346" alt="image" src="https://user-images.githubusercontent.com/50918959/170817740-33ce9e78-4061-4061-a8b7-4d8cf059a800.png">
Тому я в HTML залишила search-input-big